src/index.d.ts

```diff
@@ -265,7 +265,7 @@ export interface Options {
 	/** Attach a hook that is invoked after a vnode has rendered. */
 	diffed?(vnode: VNode): void;
 	event?(e: Event): any;
-	requestAnimationFrame?: typeof requestAnimationFrame;
+	requestAnimationFrame?(cb: () => void): void;
```

Comment:
Should probably be `any` as mentioned here https://github.com/preactjs/preact/issues/3160#issuecomment-856254254

Reply:
```suggestion
	requestAnimationFrame?(cb: () => any): void;
```

Reply:
or `CallableFunction`:

```suggestion
	requestAnimationFrame?(cb: CallableFunction): void;
```

Reply:
TS Playground link for both of the above:
https://www.typescriptlang.org/play?noUnusedParameters=true&strict=true&noImplicitOverride=true#code/JYOwLgpgTgZghgYwgAgPIAczAPYgM7IDeAUMmcgPQXJQQCOArhHmAIIjAC2cWuAYlDicIAfgAUCAEYAuZGICUyALwA+ZADdswACbzZmnaXJUa9Ji3ZceOEAKGiJMuYtXI4IAJ563no2VqMzGwc3Ly2gsLiUrIAwnAANvFwkvEQfAwgCGHeBtrEAL7ExAi4LMjYmDZ4shhhBEpE+QDcRRV1AHQB5sFWYXbCyqaBFiHW-BEQLcRtVZ1mQZahNv0oDRIJ8ZKIANayYgAesu5eymrHLmok5MgIG1sI22IAjPItza2VpXPDPUvj9oN1ol7rs5IdkCAGJxJNALhCoTCoHCruRbsCds9Xn5TGAGFAQMgnm8WkA